### DISCUSSION TITLE
Provide option to select Block Sets for Override feature

### DISCUSSION THREAD

**Discussion by smayer97** [proginosko/LeechBlockNG#367] [Ideas]
With the option to select the amount of time for the Override, please provide a selectable list of the block sets to choose, ideally only showing the block sets which have the Allow Override option selected. 

https://www.tapatalk.com/groups/leechblock/viewtopic.php?p=2096#p2096

**Comment by shipurjan** [comment]
I made a PR for it here:
https://github.com/proginosko/LeechBlockNG/pull/673
